Guardfile

```diff
@@ -38,31 +38,48 @@ group :red_green_refactor, halt_on_fail: true do
+    watch('app/controllers/v2/reservations_controller.rb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/controllers/public/people_controller.rb') { 'spec/features/people_registration_spec.rb' }
+
     watch('app/views/v2/event_invitations/new.html.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
+    watch('app/views/v2/event_invitations/new.html.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
     watch('app/views/v2/event_invitations/_form.html.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
+    watch('app/views/v2/event_invitations/_form.html.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
+    watch('app/views/v2/reservations/new.html.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/views/v2/reservations/_form.html.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
     watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
     watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
-
-    watch('spec/features/person_responds_to_interview_invitation_spec.rb')
-    watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-
-    watch('app/mailers/event_invitation_mailer.rb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/models/v2/reservation.rb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/controllers/v2/reservations_controller.rb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/views/v2/reservations/new.html.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/views/v2/reservations/_form.html.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
+    watch('app/views/public/people/new.html.erb') { 'spec/features/people_registration_spec.rb' }
+    watch('app/views/public/people/_form.html.erb') { 'spec/features/people_registration_spec.rb' }
   end
```

Comment:
Yeah, I agree it's pretty unwieldy. That's something I've been debating each time I update this, as well. A lot of the individual entries are for feature specs which have non-conventional naming, and so are hard to use patterns with. Given the time constraints, feature specs are the only coverage for certain files, and so the only things that run on guard when updating them. So without the individual entries, with these files it's like not using guard at all. I end up running the feature specs by hand after updating them.

I also really like getting immediate feedback from all the features affected when changing a  dependency.

Maybe we can find some middle ground on how to do this, conventions for naming or file org we could use to match patterns against?

@cromulus any thoughts on this?